test/test_helper.rb

```diff
@@ -1,4 +1,6 @@
 $:.unshift File.expand_path('../lib', File.dirname(__FILE__))
 require 'ifconfig'
-require 'test/unit'
+require 'minitest/autorun'
 require 'pp'
+
+Test = Minitest
```

Comment:
Sorry, this was to run unit tests on my local machine. Feel free to drop changes to test_helper.rb.